README

```diff
@@ -94,8 +112,26 @@ By default `make install` will install into `/usr/local`, for changing the path
     dependencies: *pkg-config*(build-time), *libnl-3* and *libnl-genl-3*
     default: *check*
 
-## Usage
 
+## Runtime dependencies:
+`htop` has a set of fixed minimum runtime dependencies, which is kept as minimal as possible:
+* `ncurses` libraries for terminal handling (wide character support).
+
+### Runtime optional dependencies:
+`htop` has a set of fixed optional dependencies, depending on build/configure option used:
+* `libdl`, if not building static and support for some of the optional libraries is enabled, is always required when support for to optionally load dependencies (i.e. `libsensors`, `systemd`) is present.
+* `libcap`, user-space interfaces to the POSIX 1003.1e, is always required when `--enable-capabilities` was used to configure `htop`.
+* `libsensors`, readout of temperatures and CPU speeds, is optional even when `--enable-sensors` was used to configure `htop`.
+* `systemd` is optional when `--enable-static` was not used to configure `htop` (Linux only). If building statically and `libsystemd` is not found by `configure` support for the SystemD meter is disabled entirely.
+
```

Comment:
* `libpcp`, allows building the optional `pcp-htop` binary with Performance Co-Pilot support when --enable-pcp is used to configure.